components/zcash_protocol/Cargo.toml

```diff
@@ -11,7 +11,7 @@ repository.workspace = true
 readme = "README.md"
 license.workspace = true
 edition.workspace = true
-rust-version = "1.70"
+rust-version.workspace = true
```

Comment:
Is the MSRV bump here required? We had opted not to bump past 1.70 in the previous update in order to avoid requiring a semver major version bump; since this looks like it should be additive, it'd be nice to retain an MSRV of 1.70 if we don't explicitly need features from the newer MSRV. Nonblocking, but the semver bump (and the additions) should be noted in the CHANGELOG(s).